caproto/server/records.py

```diff
@@ -727,11 +695,6 @@ class AcalcoutFields(RecordFieldGroup):
         doc='Number of Elements',
         read_only=True)
     _link_parent_attribute(display_precision, 'precision')
-    _link_parent_attribute(hihi_alarm_limit, 'upper_alarm_limit')
-    _link_parent_attribute(high_alarm_limit, 'upper_warning_limit')
-    _link_parent_attribute(low_alarm_limit, 'lower_warning_limit')
-    _link_parent_attribute(lolo_alarm_limit, 'lower_alarm_limit')
-    _link_parent_attribute(low_operating_range, 'lower_ctrl_limit')
```

Comment:
@klauer This was missing the high_operating_range links, I assumed that was an error in the initial generation?

Reply:
~fully~ 99.9% auto-generated - I did the links manually, and of course I screwed them up! Great catch.